<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>algorithm1</class>
 <widget class="QMainWindow" name="algorithm1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1357</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>算法演示 V1.0.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>寻找路径0-9</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>651</width>
      <height>651</height>
     </rect>
    </property>
    <property name="title">
     <string>深度优先搜索</string>
    </property>
    <widget class="QLabel" name="dfs">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>561</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>590</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>590</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>复位</string>
     </property>
    </widget>
    <widget class="QLabel" name="way">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>651</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>611</width>
      <height>651</height>
     </rect>
    </property>
    <property name="title">
     <string>广度优先搜索</string>
    </property>
    <widget class="QLabel" name="bfs">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>551</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="que">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>441</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>590</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>复位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>590</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
